### ISSUE TITLE
Test in virtualenv

### ISSUE THREAD

**Issue by reconmaster** [jorgenschaefer/elpy#504]
When attempting to run a nosetest with `C-c C-t` in a virtualenv, the nosetest fails saying a module that is installed only in the virtualenv does not exist. Running the same test from the command line in that same virtualenv works fine.

I thought this might be related to Issue [497](https://github.com/jorgenschaefer/elpy/issues/497), but there seems to be nothing wrong with the environmental variables, ex. `VIRTUALENVWRAPPER_HOOK_DIR=$HOME/.virtualenvs`. When using the interpreter launched in elpy in the in venv, everything works fine as well.

It is apparently only the test runner that is not using the site-packages for the venv it is supposed to be working on. The traceback is referencing the site-packages from the system outside of the virtual environment.

**Comment by jorgenschaefer** [comment]
Thank you for the report!
Which command is being run for the tests? You say you're using nosetests, then it should run the nosetests script from the virtualenv. Does it do that, or does it use some globally-installed one?

**Comment by reconmaster** [comment]
Thanks for getting back to me. As it was running the local nosetest I had installed in `.local/bin` from using `pip install --user` for packages such as virtualenvwrapper on systems where I don't have admin. I tracked it down to have `export PATH=~/.local/bin:$PATH` in my dotfiles. Changing the order appears to fix things. I'm guessing pyenv sets the `PATH` in the system process before my dotfiles then append my custom path into the system-process. Thanks again for your help.
-A
